static/app/components/searchQueryBuilder/utils.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+  filterTypeConfig,
+  interchangeableFilterOperators,
+  type TermOperator,
+  type Token,
+  type TokenResult,
+} from 'sentry/components/searchSyntax/parser';
+
+export function getValidOpsForFilter(
```

Comment:
This is nearly the same as a function in SmartSearchBar. Just copying it since that component will go away in the future